### PR TITLE
Workflow to generate and build Ui.

### PR DESCRIPTION
**Pull request by mohsinenar**
- **Workflow to generate and build Ui.**

### CODE REVIEW

File: .github/workflows/python-publish.yml [hunk 1]
```diff
@@ -11,59 +11,69 @@ name: Upload Python Package
...
+#        run: echo "TAG=$(echo $GITHUB_REF | cut -c12-17)" >> $GITHUB_ENV
+#      - name: Build image and push to ostorlab Container Registry
+#        uses: docker/build-push-action@v2
+#        with:
+#          # relative path to the place where source code with Dockerfile is located
+#          context: .
+#          tags: |
+#            ostorlab/oxo:${{env.TAG}}
+#            ostorlab/oxo:latest
+#          push: true
```
Comment (rabsondev) [inline .github/workflows/python-publish.yml:79]: Needed?

File: .github/workflows/python-publish.yml [hunk 2]
```diff
@@ -11,16 +11,26 @@ name: Upload Python Package
...
+      with:
+        node-version: '18.0.0'
+    - name: Install Node dependencies
+      run: npm install
+      working-directory: src/oxotitan
+    - name: Generate Nuxt
+      run: npm run generate
+      working-directory: src/oxotitan
+    - name: Copy Nuxt output
+      run: cp -r src/oxotitan/.output/public src/ostorlab/ui/static_files
```
Comment (3asm) [inline .github/workflows/python-publish.yml:33]: /static

File: .github/workflows/release_linux_mac_bin.yaml [hunk 3]
```diff
@@ -16,6 +16,19 @@ jobs:
...
+      with:
+        node-version: '18.0.0'
+    - name: Install Node dependencies
+      run: npm install
+      working-directory: src/oxotitan
+    - name: Generate Nuxt
+      run: npm run generate
+      working-directory: src/oxotitan
+    - name: Copy Nuxt output
+      run: cp -r src/oxotitan/.output/public src/ostorlab/ui/static_files
```
Comment (3asm) [inline .github/workflows/release_linux_mac_bin.yaml:31]: /static

**Review by 3asm — CHANGES_REQUESTED** [review]

**Comment by 3asm** [comment]
Please fix workflow.

**Review by 3asm — APPROVED** [review]

**Pull request merged**